app/src/main/java/com/keylesspalace/tusky/components/systemnotifications/NotificationHelper.java

```diff
@@ -747,29 +716,38 @@ private static void setSoundVibrationLight(AccountEntity account, NotificationCo
         }
     }
 
-    private static String wrapItemAt(StatusBarNotification notification) {
-        return StringUtils.unicodeWrap(notification.getNotification().extras.getString(EXTRA_ACCOUNT_NAME));//getAccount().getName());
+    private static String joinNames(Context context, List<StatusBarNotification> notifications1, List<Notification> notifications2) {
+        List<String> names = new ArrayList<>(notifications1.size() + notifications2.size());
+
+        for (StatusBarNotification notification: notifications1) {
+            names.add(notification.getNotification().extras.getString(EXTRA_ACCOUNT_NAME));
+        }
+
+        for (Notification notification : notifications2) {
+            names.add(notification.getAccount().getName());
+        }
+
+        return joinNames(context, names);
     }
 
     @Nullable
-    private static String joinNames(Context context, List<StatusBarNotification> notifications) {
-        if (notifications.size() > 3) {
-            int length = notifications.size();
-            //notifications.get(0).getNotification().extras.getString(EXTRA_ACCOUNT_NAME);
+    private static String joinNames(Context context, List<String> names) {
+        if (names.size() > 3) {
+            int length = names.size();
             return String.format(context.getString(R.string.notification_summary_large),
-                    wrapItemAt(notifications.get(length - 1)),
```

Comment:
I think the `String.format` is unnecessary? Format args can be passed to `context.getString` directly

Reply:
ok